modules/core/03-connection/keeper/verify.go

```diff
@@ -364,16 +364,15 @@ func (k Keeper) VerifyNextSequenceRecv(
 	return nil
```

Comment:
the diff in this file is quite confusing, the following functions were removed

- `VerifyChannelUpgradeSequence`
- `VerifyChannelUpgradeTimeout`